app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -12,11 +12,13 @@ const val EXPLORATION_ACTIVITY_TOPIC_ID_ARGUMENT_KEY = "ExplorationActivity.expl
 class ExplorationActivity : InjectableAppCompatActivity() {
   @Inject
   lateinit var explorationActivityPresenter: ExplorationActivityPresenter
+  var explorationId: String = ""
```

Comment:
remove this variable and directly pass intent to `handleOnCreate`

or make this lateinit

Reply:
To shorten I created variable. I will make it lateinit.